frontend/src/components/GoesData/BrowseTab.tsx

```
@@ -75,6 +75,90 @@ function buildFilterParams(
+  setCollectionFrameIds: (ids: string[]) => void;
+  setShowAddToCollection: (v: boolean) => void;
+  setTagFrameIds: (ids: string[]) => void;
+  setShowTagModal: (v: boolean) => void;
+  setCompareFrames: (v: [GoesFrame, GoesFrame] | null) => void;
+}>) {
+  if (selectedIds.size === 0) return null;
+
+  return (
+    <div className="hidden md:contents">
+      <button onClick={() => { if (globalThis.confirm(`Delete ${selectedIds.size} frame(s)? This action cannot be undone.`)) deleteMutation.mutate([...selectedIds]); }} aria-label="Delete selected frames"
+        className="flex items-center gap-1 px-3 py-1.5 text-xs bg-red-600/20 text-red-400 rounded-lg hover:bg-red-600/30 transition-colors min-h-[44px]">
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)